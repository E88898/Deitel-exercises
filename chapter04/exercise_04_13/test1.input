287
13
-1

