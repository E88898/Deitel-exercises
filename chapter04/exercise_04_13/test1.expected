MPG this tankful: 22.0769
Total MPG: 22.0769
